### PR TITLE
feat(docs/3.3.x): Set docs version numbers dynamically from file

### PR DESCRIPTION
**Pull request by big-r81**
Load documentation version from local version.mk file.

### CODE REVIEW

File: src/docs/src/conf.py [hunk 1]
```diff
@@ -29,9 +30,13 @@
...
-# should be over-written using rebar-inherited settings
-version = "3.3"
-release = "3.3.0"
+# load version numbers from version.mk
+version_file = Path().absolute().joinpath('../../../version.mk').resolve()
+with open(version_file) as file:
+    props = dict(line.strip().split('=', 1) for line in file)
+
+version = f"{props['vsn_major']}.{props['vsn_minor']}" #"3.3"
+release = f"{props['vsn_major']}.{props['vsn_minor']}.{props['vsn_patch']}" #"3.3.3"
```
Comment (jaydoane) [inline src/docs/src/conf.py:39]: Do you want to leave the comments in place, which will become obsolete as soon as there's a new release?

**Review by jaydoane — APPROVED** [review]

File: src/docs/src/conf.py (hunk 1 above)
Comment (big-r81) [reply src/docs/src/conf.py]: I have removed it. In the beginning it was meant to be easier to read, but it might just lead to confusion.

**Comment by big-r81** [comment]
Getting this python-black error, which I can’t reproduce locally. Should I merge it anyway or do you have any tips?

File: src/docs/src/conf.py [hunk 2]
```diff
@@ -29,9 +30,13 @@
 
 nitpicky = True
 
-# should be over-written using rebar-inherited settings
-version = "3.3"
-release = "3.3.0"
+# load version numbers from version.mk
+version_file = Path().absolute().joinpath('../../../version.mk').resolve()
```
Comment (jaydoane) [inline src/docs/src/conf.py:34]: I'm seeing `black` complain about the single quotes around `'../../../version.mk'`. 

`make python-black-update` changes it to double quotes.

File: src/docs/src/conf.py [hunk 3]
```diff
@@ -29,9 +30,13 @@
...
+with open(version_file) as file:
+    props = dict(line.strip().split('=', 1) for line in file)
```
Comment (jaydoane) [inline src/docs/src/conf.py:36]: `black` also complains about the single quotes around the equal sign. Try changing to `"="`.

**Comment by big-r81** [comment]
**REMINDER**

`erlfmt` IS NOT `python-black`.
`erlfmt` IS NOT `python-black`.
`erlfmt` IS NOT `python-black`.
`erlfmt` IS NOT `python-black`.

Thx, @jaydoane 😉 🙄

**Pull request merged**